Chart.yaml

```diff
@@ -23,7 +23,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.0
+version: 0.1.1
```

Comment:
Please do not change `version`, it follows kubediag release.

Reply:
Helm Chart has two kind of version actually, one is `appVersion` which is the kubediag release version that you mentioned, another is chart `version` which is modified here, and should follow the semver after we change the chart.

Reply:
Yes, you are right.